### PR TITLE
Onboarding WordPress importer: content everything flow

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

* Improved **content-chooser component** (disable Continue button until the info from the server is received)
* Implemented **Import Everything** flow (extended the class from the migration module with overwritten render UI logic)
* Implemented **Confirm screen** with the accompanying modal
* Moved common styles to the parent import styles.css file

#### Testing instructions

* Go to `/start/importer/capture?siteSlug={SLUG_ATOMIC}`, target website should be under the Business plan
* Enter the source website URL, example: `amused-swan.jurassic.ninja` (it has jetpack installed)
* From the ready screen, press the "Import your content" button
* Then select the "Everything" option by pressing the "Continue" button
* And finally, press the "Start import" button and confirm the modal
* Check if there is a proper Progress screen with a Hooray screen at the end

#### Screenshots
Markup on 2022-01-19 at 13:22:55
Markup on 2022-01-19 at 13:23:50
Markup on 2022-01-19 at 13:24:23
Markup on 2022-01-19 at 13:24:55
Markup on 2022-01-19 at 13:25:30

#### Known issues:
* #60237
* #60238
* #60241

Related to #59042

### CODE REVIEW

File: client/signup/steps/import-from/wordpress/import-everything/confirm-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+const ConfirmModal: React.FunctionComponent< Props > = ( data ) => {
+	const { __ } = useI18n();
+	const { siteSlug, onConfirm, onClose } = data;
+
+	return (
+		<Modal
+			className={ classnames( 'components-modal-new__frame', 'import__confirm-modal' ) }
+			title={ __( 'Import and replace everything?' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import-from/wordpress/import-everything/confirm-modal.tsx:22]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Import and replace everything on this site?' )` *ES Score: 7*

File: client/signup/steps/import-from/wordpress/import-everything/index.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+					<div className={ classnames( 'import-layout', 'import__site-mapper-border' ) }>
+						<div className={ classnames( 'import-layout__column' ) }>
+							<div
+								className={ classnames(
+									'import-layout__column',
+									'import__site-mapper-header',
+									'import__site-mapper-border'
+								) }
+							>
+								{ __( 'Original site' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import-from/wordpress/import-everything/index.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Original Site' )` *ES Score: 10*

File: client/signup/steps/import-from/wordpress/import-everything/confirm.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+	return (
+		<>
+			<div className={ classnames( 'import__import-everything' ) }>
+				<div className={ classnames( 'import__site-mapper' ) }>
+					<div className={ classnames( 'import-layout', 'import__site-mapper-border' ) }>
+						<div className={ classnames( 'import-layout__column' ) }>
+							<div
+								className={ classnames( 'import__site-mapper-header import__site-mapper-border' ) }
+							>
+								{ __( 'Original site' ) }
```
Comment (a8ci18n) [inline client/signup/steps/import-from/wordpress/import-everything/confirm.tsx:41]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Original Site' )` *ES Score: 10*

**Review by vishnugopal — COMMENTED** [review]
I get these error when I hit Everything:

```
index.tsx:156 
        
       Uncaught TypeError: Cannot read properties of undefined (reading 'URL')
    at ImportEverything.renderMigrationConfirm (index.tsx:156:33)
    at ImportEverything.render (index.tsx:232:21)
    at finishClassComponent (react-dom.development.js:17485:31)
    at updateClassComponent (react-dom.development.js:17435:24)
    at beginWork (react-dom.development.js:19073:16)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:16)
    at invokeGuardedCallback (react-dom.development.js:4056:31)
    at beginWork$1 (react-dom.development.js:23959:7)
    at performUnitOfWork (react-dom.development.js:22774:12)
renderMigrationConfirm @ index.tsx:156
render @ index.tsx:232
finishClassComponent @ react-dom.development.js:17485
updateClassComponent @ react-dom.development.js:17435
beginWork @ react-dom.development.js:19073
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
beginWork$1 @ react-dom.development.js:23959
performUnitOfWork @ react-dom.development.js:22774
workLoopSync @ react-dom.development.js:22702
renderRootSync @ react-dom.development.js:22665
performSyncWorkOnRoot @ react-dom.development.js:22288
eval @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
scheduleUpdateOnFiber @ react-dom.development.js:21888
enqueueSetState @ react-dom.development.js:12467
Component.setState @ react.development.js:365
eval @ section-migrate.jsx:194
eval @ section-migrate.jsx:343
Promise.then (async)
eval @ section-migrate.jsx:298
componentDidMount @ section-migrate.jsx:394
commitLifeCycles @ react-dom.development.js:20658
commitLayoutEffects @ react-dom.development.js:23421
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
commitRootImpl @ react-dom.development.js:23146
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
commitRoot @ react-dom.development.js:22985
performSyncWorkOnRoot @ react-dom.development.js:22324
eval @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
flushPassiveEffectsImpl @ react-dom.development.js:23615
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23442
eval @ react-dom.development.js:23319
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
react-dom.development.js:20085 
        
       The above error occurred in the <ImportEverything> component:

    at ImportEverything (app:///./signup/steps/import-from/wordpress/import-everything/index.tsx:63:5)
    at eval (app:///../packages/i18n-calypso/src/localize.js:29:67)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at WordpressImporter (app:///./signup/steps/import-from/wordpress/index.tsx:38:76)
    at div
    at div
    at div
    at div
    at StepWrapper (app:///./signup/step-wrapper/index.jsx:32:1)
    at eval (app:///../packages/i18n-calypso/src/localize.js:29:67)
    at ImportOnboardingFrom (app:///./signup/steps/import-from/index.tsx:61:5)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at div
    at div
    at div
    at div
    at Signup (app:///./signup/main.jsx:143:5)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at div
    at div
    at div
    at Layout (app:///./layout/index.jsx:242:1)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at WithCurrentRoute (app:///./components/route/index.js:44:30)
    at MomentProvider (app:///./components/localized-moment/provider.js:27:5)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at Provider (app:///../node_modules/react-redux/es/components/Provider.js:18:20)
    at QueryClientProvider (app:///../node_modules/react-query/es/react/QueryClientProvider.js:38:21)
    at RouteProvider (app:///./components/route/index.js:22:3)
    at I18nProvider (app:///../node_modules/@wordpress/react-i18n/build-module/index.js:68:5)
    at LocaleProvider (app:///../packages/i18n-utils/src/locale-context.tsx:19:3)
    at CalypsoI18nProvider (app:///./components/calypso-i18n-provider/index.tsx:21:3)
    at ProviderWrappedLayout (app:///./controller/index.web.js:60:3)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
```

My URL is this:
```
http://calypso.localhost:3000/start/from/importing/wordpress?from=https%3A%2F%2Famused-swan.jurassic.ninja%2F&to=vishnugopala8ctest200.wordpress.com&option=everything
```

**Comment by bogiii** [comment]
In the instructions, I've proposed to use the `amused-swan.jurassic.ninja` website with the installed jetpack. The site exists and is created by me and I have permission on both source and target sites.

That's the reason why you got the error!
Let's say that we caught another bug that I will cover later. The bug ticket: #60342

Please try again with your own website with an installed jetpack, and for that, you can use the `jurassic.ninja`.

@vishnugopal

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7098982

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7098982).

**Pull request merged**